2_basic_chisel.ipynb

```diff
@@ -60,18 +60,318 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "collapsed": false,
-    "scrolled": true
-   },
-   "outputs": [],
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "\u001b[32mimport \u001b[39m\u001b[36m$ivy.$                                                 \n",
```

Comment:
We should probably avoid checking in outputs

Reply:
Agreed, I'll fix it.